### PR TITLE
Enable Appveyor

### PR DESCRIPTION
**Pull request by cadecairos**
Adds appveyor.yml to configure running tests on Windows

fixes https://github.com/MozillaFoundation/mofo-devops/issues/510

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
     "build:common": "shx rm -rf dest && npm run build:env && npm run fetch && run-p build:images build:pug build:sass && npm run build:healthcheck && npm run build:contribute",
     "build:contribute": "shx cp contribute.json dest/",
     "build:healthcheck": "node scripts/build-healthcheck.js > dest/healthcheck.html",
-    "build:images": "shx rm -rf dest/_images && shx cp -r source/images dest/_images",
+    "build:images": "shx rm -rf dest/_images && shx mkdir -p dest/ && shx cp -r source/images dest/_images",
```
Comment (cadecairos) [inline package.json]: I honestly don't know if this fixes the random error, or if I got lucky...

File: package.json [hunk 2]
```diff
@@ -28,8 +28,8 @@
     "test:eslint": "eslint --config ./.eslintrc.yaml scripts/**/*.js source/js/**/*.js source/js/components/**/*.jsx",
     "test:json": "ajv validate -s source/json/upcoming.schema.json -d source/json/upcoming.json && ajv validate -s source/json/highlights.schema.json -d source/json/highlights.json",
     "test:pulled-json": "ajv validate -s source/json/people.schema.json -d source/json/temp/people.json && ajv validate -s source/json/news.schema.json -d source/json/temp/news.json",
-    "test:pug": "pug-lint source/pug/**/*.pug",
-    "test:scss": "stylelint 'source/sass/main.scss' --syntax scss",
+    "test:pug": "pug-lint source/pug/partials/ source/pug/templates/ source/pug/partials/",
```
Comment (cadecairos) [inline package.json]: Windows doesn't support Glob patterns.

File: package.json [hunk 3]
```diff
@@ -28,8 +28,8 @@
...
+    "test:scss": "stylelint source/sass/main.scss --syntax scss",
```
Comment (cadecairos) [inline package.json]: `shx` was double quoting the argument on windows, resulting in `'\'source/sass/main.scss\''`

File: package.json (hunk 1 above)
Comment (gvn) [inline package.json]: Revisiting this I can see there's a slightly DRY-er way. Can you try this?

https://github.com/mozilla/network/pull/542

**Comment by cadecairos** [comment]
@gvn Your changes worked, nice!

**Pull request merged**